rpm/SPECS/iotasigfox.spec

```diff
@@ -0,0 +1,181 @@
+Summary: Sigfox IoT Agent
```

Comment:
License header missing.

Reply:
Added license header in https://github.com/telefonicaid/sigfox-iotagent/pull/32/commits/c4b64018e59a70afef77312d670c787539d6d009